api/src/main/java/com/codeforcommunity/rest/subrouter/BlocksRouter.java

```diff
@@ -93,4 +101,15 @@ private void handleResetRoute(RoutingContext ctx) {
 
     end(ctx.response(), 200, JsonObject.mapFrom(response).encode());
   }
+
+  private void handleGetReserved(RoutingContext ctx) {
+    JWTData userData = ctx.get("jwt_data");
+
+    boolean includeDone =
```

Comment:
Use a method in RestFunctions for this that can do null checking properly (if there's not one for query params then create it)

Reply:
Done